docs/guide/acquiring_data/database.rst

```diff
@@ -12,37 +12,31 @@ database is used) or a local or remote database server (see the SQLAlchemy
 documentation for a `list of supported databases
 <http://docs.sqlalchemy.org/en/rel_0_8/dialects/>`_)
 This makes it possible to fetch required data from the local database
-instead of downloading it again from a remote server. The package
-:mod:`sunpy.database` was developed as part of `Google Summer of Code
-(GSOC) 2013
+instead of downloading it again from a remote server.
+
+The package :mod:`sunpy.database` was developed as part of
+`Google Summer of Code (GSOC) 2013
 <http://www.google-melange.com/gsoc/homepage/google/gsoc2013>`_.
 
 1. Connecting and initializing the database
 -------------------------------------------
-To start a connection to an existing or a new database, instantiate
-a :class:`Database` object.
+To start a connection to an existing or a new database, create
+a :class:`Database` object:
 
     >>> from sunpy.database import Database
     >>> database = Database('sqlite:///sunpydata.sqlite')
 
 The database object in our example above connects to a new SQLite database with
 the file name "sunpydata.sqlite" in the current directory.
 
-The first parameter of ``Database`` receives one mandatory argument:
-a URL which describes how to connect to the database. This value is
-directly passed to :func:`sqlalchemy.create_engine`. The supported
+The first parameter of :class:`Database` receives one mandatory argument:
+a URL which describes how to connect to the database. The supported
 format of this URL is described by the documentation of
-:func:`sqlalchemy.create_engine` as follows:
-
-    "The string form of the URL is
-    ``dialect+driver://user:password@host/dbname[?key=value..]``, where
-    dialect is a database name such as ``mysql``, ``oracle``, ``postgresql``,
-    etc., and driver the name of a DBAPI, such as ``psycopg2``,
-    ``pyodbc``, ``cx_oracle``, etc."
+:func:`sqlalchemy.create_engine`.
```

Comment:
Would not knowing how to structure the url string be important? 

Maybe that's only important if you already have some sort of remote server with the database on.

Reply:
I figured the description is in `sqlalchemy.create_engine`, so if you want to find out it's fairly easy to click on the link.

Reply:
That's fine by me.